src/app/pages/details-texts/details-texts.component.ts

```diff
@@ -1,14 +1,16 @@
-import { Component, OnInit } from "@angular/core";
+import { Component, OnInit, ViewEncapsulation } from "@angular/core";
 import { GetDataService } from "../../services/get-data/get-data.service";
 import { DomSanitizer } from "@angular/platform-browser";
 import { HandleBreadcrumbsService } from "../../services/handle-breadcrumbs/handle-breadcrumbs.service";
-import { Router } from "@angular/router";
+import { Router, ActivatedRoute, ParamMap } from "@angular/router";
 import { composedPath } from "../../../utils/utils";
 import { DIRECTION, PANEL_TYPE, SESSION_KEYS } from "../../../utils/consts";
 
 @Component({
   selector: "app-details-texts",
-  templateUrl: "./details-texts.component.html"
+  templateUrl: "./details-texts.component.html",
+  styleUrls: ["../details/details.component.scss"],
```

Comment:
Hm, so were we not applying any styling to this before?

Reply:
What was happening before is that the styles were inherited from previous components. For example, as the user searched for a term the 'details' component would be rendered as the user clicked through the various results tables. Then when the user got to the 'details-text' component it would inherit the styles that came before. 

However, when the user enters a manual url they are skipping the components that came before so it's necessary to explicitly redefine the styles. We could copy and paste the styles from the 'details' component and make a new css file for 'details-texts', but I decided to keep one file which is why I reference it from a different folder (line 12).

Hope that makes sense, happy to chat about this further if needed.